templates/web/base/report/_inspect.html

```diff
@@ -16,6 +16,9 @@ <h2 class="inspect-form-heading">[% loc('Inspect report') %]</h2>
         <p>
           <strong>[% loc('Report ID:') %]</strong>
           <span class="js-report-id">[% problem.id %]</span>
+          [% IF c.user_exists AND c.cobrand.admin_allow_user(c.user) AND c.user.has_permission_to('report_edit', problem.bodies_str_ids) %]
+            (<a href="[% c.uri_for( c.controller('Admin').action_for('report_edit'), problem.id ) %]">[% loc('admin') %]</a>)
```

Comment:
I think this can be `[% c.uri_for_action( 'admin/report_edit', problem.id ) %]` or similar